proxy/fyde_proxy_bm_vm.md

```diff
@@ -1,60 +1,82 @@
 # Install Fyde Proxy in Bare Metal / Virtual Machine
 
 - Supported distributions:
+  - CentOS 7
 
-  - [CentOS 7](##centos-7)
+- Instalation requires a valid [Proxy Enrollment Link](../console/configurations/add_proxy.md#adding-a-proxy)
 
-## Centos 7
+- Choose [**Install script**](##install-script) or [**Manual steps**](##manual-steps) to proceed
+
+## Install script
+
+- Please note that the steps below will execute scripts obtained externally
+- We advise to inspect the content before execution
+
+1. Download and execute script
+
+    ```sh
+    curl -fsSL https://url.fyde.me/install-fyde-proxy-centos \
+        -o install-fyde-proxy-docker.sh
+    sudo bash install-fyde-proxy-docker.sh
+    ```
+
+## Manual steps
 
 1. Install Software Collections (SCL) repository
 
     ```sh
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
-    yum -y install centos-release-scl yum-utils
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-SCLo
+    sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```

Comment:
@GMartinez-Sisti for those running this in docker (and copy pasting) wouldn't it be better NOT to have `sudo` here? We could require before that the user is root

Reply:
In this document (proxy/fyde_proxy_bm_vm.md ) we have installation steps for Bare Metal / Virtual Machine. For docker steps we have another document -> proxy/fyde_proxy_docker.md .